lib/shared/bucketing-assembly-script/assembly/types/protobuf/as-generated/DVCUser_PB.ts

```diff
@@ -0,0 +1,156 @@
+// Code generated by protoc-gen-as. DO NOT EDIT.
```

Comment:
Do we want to include these generated files? or should we gitignore them?

Reply:
thoughts @ajwootto ?

Reply:
not fussed either way, probably makes sense to include them since otherwise you'd get type errors when cloning the repo until you build it

Reply:
I removed them, as it seemed like generated files shouldn't be part of the repo.

Reply:
ahh I think I need to add a brew install script to install `protoc`

Reply:
FWIW, most examples of protobuf I've seen check in the generated files. That might be biased towards Go though, where it's more common to have repos that build with just the base tools right after cloning.
Also means you don't need protoc installed just to build the repo, only if you're changing the protobuf definitions.

Reply:
happy to add them back in if you think that makes sense.